packages/components/src/Card/tests/Card.test.tsx

```diff
@@ -1,6 +1,6 @@
 import React from "react";
 import renderer from "react-test-renderer";
-import { fireEvent, render } from "@testing-library/react";
+import { fireEvent, render, screen } from "@testing-library/react";
```

Comment:
```suggestion
import { fireEvent, render } from "@testing-library/react";
```

Reply:
@shreyavi screen wasn't removed, you shouldn't need to get screen from the package itself, and should instead destructure the render function

Reply:
I am so sorry about that, just removed it